.github/workflows/ci.yml

```
@@ -51,232 +49,197 @@ jobs:
       - name: Cache Rust dependencies
         uses: actions/cache@v3
         with:
           path: target
           key: ${{ runner.OS }}-build-${{ hashFiles('**/Cargo.lock') }}
           restore-keys: |
             ${{ runner.OS }}-build-
-      - uses: actions-rs/toolchain@v1
-      - run: rustup component add rustfmt
-      - uses: actions-rs/cargo@v1
+      - name: cargo check
+        uses: actions-rs/cargo@v1
```

actionlint (1.7.7)

the runner of "actions-rs/cargo@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)